app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html

```diff
@@ -135,7 +136,7 @@
     <h1 class="heading-xlarge">@title</h1>
     @if(!isPeriodOfAdmin) {
       <span class="heading-large" style="font-weight: normal;">
-        @messages(s"$key.taxOwed")
+        @messages(s"$key.${if(isAmend) "revisedLiability" else "taxOwed"}")
```

Comment:
I think its good to extract this logic into a variable like
`@x = @{ if(isAmend) "revisedLiability" else "taxOwed" }`
then
@messages(s"$key.$x")

Reply:
ok, that's done - better to have less logic in key names!